### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jseldess**
- Improve Quickstart steps
- Add link to troubleshooting docs and Stack Overflow

### CODE REVIEW

**Comment by tamird** [comment]
Without reviewing for content, could you please wrap these lines?

File: README.md [hunk 1]
```diff
@@ -46,11 +46,11 @@ For guidance on installation, development, deployment, and administration, see o
 
 1. [Start a local cluster](https://www.cockroachlabs.com/docs/start-a-local-cluster.html) and talk to it via the [built-in SQL client](https://www.cockroachlabs.com/docs/use-the-built-in-sql-client.html).
 
-1. [Secure the cluster](https://www.cockroachlabs.com/docs/secure-a-cluster.html) with TLS encryption.
+1. [Learn more about CockroachDB SQL](https://www.cockroachlabs.com/docs/learn-cockroachdb-sql.html).
 
-1. [Learn about CockroachDB SQL](https://www.cockroachlabs.com/docs/learn-cockroachdb-sql.html).
+1. Use a PostgreSQL-compatible driver or ORM to [build an app with CockroachDB](build-an-app-with-cockroachdb.html).
```
Comment (a-robinson) [inline README.md]: You need to use the full URL here

**Review by a-robinson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jseldess) [inline README.md]: Done. Thanks for catching that.

**Comment by jseldess** [comment]
@tamird, wrapped the lines I worked on. PTAL.

**Pull request merged**